src/mame/rm/rmnimbus.h

```diff
@@ -149,6 +153,8 @@ class rmnimbus_state : public driver_device
 	void nimbus_sound_ay8910_porta_w(uint8_t data);
 	void nimbus_sound_ay8910_portb_w(uint8_t data);
 	uint8_t nimbus_joystick_r();
+	void nimbus_select_joystick0(uint8_t data);
+	void nimbus_select_joystick1(uint8_t data);
```

Comment:
Do you actually need two of these?  Couldn’t you have a single write handler across the range, and get the selected joystick from the offset?

Reply:
I will consider this while making the other change. For readability two functions might still be cleaner as there are only two.

Reply:
After trying it out I think you are right so have made this change with another commit!

Reply:
@cuavas, one thing I've noticed is that writing to address 0xa1 and 0xa3 now also changes the selected joystick.  Do you think that is a problem?  I could change the data type to uint8_t and then check for an even offset etc.

Reply:
You can either change to uint8_t like that or add the mem_mask parameter and use the byte lane selection tests like `if (ACCESSING_BITS_0_7)` / `if (ACCESSING_BITS_8_15)`.  The uint8_t is probably a little cleaner if ports 0xa1/0xa3 aren't actually used.

Reply:
Thanks for the info.  0xa1/0xa3 are not used so I've pushed another commit with the unit8_t solution so that only even addresses are used.  I think everything should be fine now.